src/components/StaticAppBar/StaticAppBar.css

```diff
@@ -19,6 +19,9 @@
 .drawerAppBar button{
     margin-top: 0px !important;
 }
+.top-menu{
+    top: -26px;
```

Comment:
Would this be better as a percent change than a hard value like this? different screens view it different. Can you test this on
- [ ] 4K Display resolution
- [ ] 2K 
- [ ] HD
- [ ] SD